misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/RawMaterialIssueScanScreen.jsx

```diff
@@ -39,6 +41,27 @@ function RawMaterialIssueScanScreen(WrappedComponent) {
       };
     }
 
+    componentDidMount() {
+      const {
+        stepProps: { huBarcode, qtyToIssue },
+      } = this.props;
+      const location = getLocation(this.props, true);
```

Comment:
pls create a function which creates the location for a manufacturing raw material issue scan and which takes specific params.